proxy/frontend/type/postgresql/src/main/java/org/apache/shardingsphere/proxy/frontend/postgresql/err/PostgreSQLErrPacketFactory.java

```diff
@@ -65,8 +71,8 @@ private static PostgreSQLErrorResponsePacket createErrorResponsePacket(final Ser
     
     @SuppressWarnings("ConstantConditions")
     private static PostgreSQLErrorResponsePacket createErrorResponsePacket(final SQLException cause) {
-        if (cause instanceof PSQLException && null != ((PSQLException) cause).getServerErrorMessage()) {
```

Comment:
The `cause` could be `PSQLException`, so we need to keep the origin logic here and add a new branch to handle `PostgreSQLException`.